homeassistant/components/nuki/__init__.py

```diff
@@ -46,6 +60,30 @@ def _get_bridge_devices(bridge: NukiBridge) -> tuple[list[NukiLock], list[NukiOp
     return bridge.locks, bridge.openers
 
 
+def _register_webhook(bridge: NukiBridge, entry_id: str, url: str) -> bool:
+    # Register HA URL as webhook if not already
+    callbacks = bridge.callback_list()
+    for item in callbacks["callbacks"]:
+        if entry_id in item["url"]:
+            if item["url"] == url:
+                return True
+            bridge.callback_remove(item["id"])
+
+    if bridge.callback_add(url)["success"]:
+        return True
+
+    return False
+
+
+def _remove_webhook(bridge: NukiBridge, entry_id: str) -> bool:
+    # Remove webhook if set
+    callbacks = bridge.callback_list()
+    for item in callbacks["callbacks"]:
+        if entry_id in item["url"]:
+            bridge.callback_remove(item["id"])
+    return True
```

Comment:
there is nothing that return False or get checked, why not just set to None?

Reply:
Changed